src/App.tsx

```diff
@@ -15,48 +15,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortFields {
+  DEFAULT = '',
+  ALPHABETICLY = 'alphabet',
+  LENGTH = 'length',
```

Comment:
- you can use upperCase only for constants
- for the enum you can use only CamelCase
```suggestion
  Default = '',
  Alphabeticly = 'alphabet',
  Length = 'length',
```